src/app/store.ts

```diff
@@ -1,10 +1,17 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
Replace `combineSlices` with `combineReducers` from `@reduxjs/toolkit` to correctly set up the Redux store.